contracts/StablecoinConverter.sol

```diff
@@ -355,6 +345,22 @@ contract StablecoinConverter is EpochTokenLocker {
         previousSolution.objectiveValue = newObjectiveValue;
     }
 
+    function getTradedAmounts(uint128 volume, Order memory order) private view returns (uint128, uint128) {
+        // ToDo: implement logic also for buyOrders
```

Comment:
nit: Just fyi, I thin the convention for the ToDos is to follow this pattern:
```js
// TODO: message
```
There's some automatic tools that helps keeping track of them and they don't recognize `ToDo`

![image](https://user-images.githubusercontent.com/2352112/66065541-e9d50000-e547-11e9-9f73-16f6df875d26.png)

By the way, I just tried this one and looks nice: https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree

Reply:
Cool. Good to know